platform/glue-gen.sh

```diff
@@ -0,0 +1,25 @@
+
+# Remove old glue flies
+rm -rf platform/src/glue
+rm -rf platform/src/glue_manual
+
+# Generate manual glue - NOTE this is a workaround until glue generates for all types
+roc glue ../roc/crates/glue/src/RustGlue.roc platform/src/glue_manual platform/main-manual-glue.roc
```

Comment:
The comment says glue doesn't generate for all types, but then the code goes ahead and uses glue! 😆 But if it doesn't work why are we using it? I think it would be good to add some more background detail to these code comments.

Reply:
So glue generation on the `mainForHost : Request -> Task Response []` only generates for types that are include in that signature namely `Request` and `Response` and their direct dependencies. Any other types which are used in other effects are not generated, so this `glue_manual` is generated by `platform/main-manual-glue.roc` which is a workaround for this issue. I don't know the exact cause of the issue but I'm am pretty sure it relates to https://github.com/roc-lang/roc/issues/5477.